sectionsUtilityTool/build.sh

```diff
@@ -1,23 +1,19 @@
 #!/bin/sh --
 
 #Helper script for use on build server.
-#
 # Intended to be run after the Maven build is successful.
-#
 
 # Debugging: -x to enable, +x to disable
 set +x
 
 timestamp=$(date +%Y%m%d%H%M%S)
-
-# configuration-files.txt contains list of files to archive.
-# Use "-C <dir>" to change to <dir> before processing remaining files.
-#tar --exclude .svn -cf target/configuration-files.${timestamp}.tar \
-# $(cat configuration-files.txt)
-
+cd sectionsUtilityTool
 cd target
-
 warFilename=$(ls *.war | head -1)
 targetFilename=$(basename ${warFilename} .war)
+#GIT_BRANCH =origin/TLUNIZIN-424 or origin/master jenkins environmental variable to get git branch
+branch=${GIT_BRANCH}
+# substring'ing  ignores string before /
+btemp=${branch:7}
+mv ${targetFilename}.war ${targetFilename}.${timestamp}.${btemp}.war
 
```

Comment:
I'd put the branch name before the timestamp so if there is a reason to sort a list of war file names they will automatically group by project not time.  This could happen if AWI keeps an archive directory of war files that have been installed.